tests/ts_simple_select_iso8601.erl

```diff
@@ -69,16 +69,18 @@
                 },
 
                 {0,
-                 {">", "2016-08-02 10:19"},
-                 {"<", "2016-08-02 10:20"}
-                },
-
-                {0,
                  {">", "2016-08-02 10:19.99"},
                  {"<", "2016-08-02 10:20"}
                 }
                ]).
 
+-define(FAIL_TESTS, [
+                {0,
```

Comment:
Overall the change looks good @korry8911. Please drop the tally here, however, since it'll just be clutter for these tests. (You'll have to drop the code that matches and ignores the tally too.)